### ISSUE TITLE
InfluxDB backup issues after updating to version 4.0.30

### ISSUE THREAD

**Issue by downdrown** [tiredofit/docker-db-backup#309]
### Summary

After updating to version 4.0.30 my InfluxDB backup is not working any longer.
From what I can see it looks like the checksum generation somehow fails and then the whole backup process just exits.
The MariaDB backup works just fine.
![checksum_generation](https://github.com/tiredofit/docker-db-backup/assets/3913600/a2eb15bb-518c-426c-b2e2-b24f48900fda)
![backup_dir](https://github.com/tiredofit/docker-db-backup/assets/3913600/5b3e7bec-c3fb-46e5-bd89-02be28243575)

### Steps to reproduce
* See attached `docker-compose.yml for env details`

### What is the expected *correct* behavior?
* InfluxDB backup is stored in the backup dir

### Relevant logs and/or screenshots
See the [attached debug output](https://github.com/tiredofit/docker-db-backup/files/13692547/_db-backup_logs.txt)

### Environment
- Image version / tag:  `4.0.30`
- Host OS: `Unraid`

<details>
<summary>Any logs | docker-compose.yml</summary>

### docker-compose.yml
```
version: "3"

volumes:
  backup-mariadb:
    external: true
  backup-influxdb:
    external: true
  backup-logs:
    driver: local
    
networks:
  db-backup:
    external: true
    
services:
  db-backup:
    image: tiredofit/db-backup:${DOCKER_IMAGE_VERSION}
    container_name: db-backup
    restart: always
    networks:
      - db-backup
    environment:
      - TIMEZONE=Europe/Vienna
      - CONTAINER_NAME=db-backup
      - CONTAINER_ENABLE_MONITORING=FALSE
      - BACKUP_JOB_CONCURRENCY=2
      - DEFAULT_CHECKSUM=MD5
      - DEFAULT_COMPRESSION=GZ
      - DEFAULT_BACKUP_INTERVAL=1440
      - DEFAULT_BACKUP_BEGIN=0100
      - DEFAULT_CLEANUP_TIME=8640
      - DB01_TYPE=influx
      - DB01_INFLUX_VERSION=2
      - DB01_HOST=${INFLUXDB_HOST}
      - DB01_NAME=ALL
      - DB01_USER=${INFLUXDB_USER}
      - DB01_PASS=${INFLUXDB_PASSWORD}
      - DB01_FILESYSTEM_PATH=/backup/influxdb
      - DB02_TYPE=mariadb
      - DB02_HOST=${MARIADB_HOST}
      - DB02_NAME=ALL
      - DB02_NAME_EXCLUDE=${MARIADB_EXCLUDED_SCHEMAS}
      - DB02_USER=${MARIADB_USER}
      - DB02_PASS=${MARIADB_PASSWORD}
      - DB02_FILESYSTEM_PATH=/backup/mariadb

    volumes:
      - backup-mariadb:/backup/mariadb
      - backup-influxdb:/backup/influxdb
      - backup-logs:/logs
```
</details>

**Comment by tiredofit** [comment]
Come on up to `4.0.32` and it Influx will start working again, was resolved yesterday from #308

**Comment by downdrown** [comment]
I'll do that, thanks!
